ui/ui_welcome.py

```diff
@@ -321,12 +321,23 @@ def on_proc_picked(self, widget_android_package):
         accept, what = editor.show()
         if accept:
             self.startup_script = what
-            self.app.get_dwarf().attach(widget_android_package.get_pid(), script=what)
+            app_name = widget_android_package.appname
+            app_pid = widget_android_package.get_pid()
+            if "\t" in app_name:
```

Comment:
@iGio90 here it's simple fix for iOS but not 100% sure Android output is the same way:

`1010\tAppName`?

Reply:
works good!